setup.py

```diff
@@ -35,6 +35,7 @@
         "tensorflow",
         "ms2deepscore",
         "gensim==3.8.0",
+        "pandas==1.2.5"
```

Comment:
Those requirements are **very** strict. We should check if we can make them more flexible.
Otherwise people will often not be able to combine MS2Query with other packages.
Would the following also work: `pandas>=1.2.5` (or if not, at least `pandas == 1.2.*`) ?
Same for gensim I guess (plus spec2vec was now changed to `gensim>=4.0.0`)

Reply:
Yes good point I agree. 
I changed the gensim to 3.8.0, since 4.0 gave errors for Spec2vec, but since this is solved by now this can indeed be changed for sure. I am not sure why the pandas was set like this, I have a vague memory that it gave errors in combination with pickle, but I will have a look if it works with pandas>=1.2.5 and otherwise make an issue addressing this problem.

Reply:
One of the commits in this pull requests is indeed named set pandas to 1.2.5 to prevent pickle load error. 
The problem was that when loading pickled files created in older pandas versions that these are not compatible with methods from this new pandas version. Since all files used for the tests are in this older pickle version, I decided to use a quick fix of changing pandas at 1.2.5.  But this has to be changed. To be fair I think it might actually be better to completely get rid of pickle and replace it with a different data storing method. I created a new issue, where this can be discussed further. #104

Reply:
Found another issue created some time ago that already suggests using a different method than pickle. see #67, so I closed #104 again